credoai/prism/compare.py

```diff
@@ -0,0 +1,64 @@
+"""Performs comparison between 2 pipelines"""
+from typing import List, Literal, Optional
+from credoai.evaluators.utils.validation import check_instance
+from credoai.evidence.containers import MetricContainer
+from credoai.lens import Lens
+
+from credoai.utils import flatten_list
+from credoai.prism.comparators.metric_comparator import MetricComparator
+
+
+class Compare:
```

Comment:
Why does this class need to exist? I thought this class was essentially what Prism was doing